src/servers/src/http.rs

```diff
@@ -112,6 +113,7 @@ pub struct HttpServer {
     shutdown_tx: Mutex<Option<Sender<()>>>,
     user_provider: Option<UserProviderRef>,
     metrics_handler: Option<MetricsHandler>,
+    http_configurator: ConfiguratorRefOption,
```

Comment:
Because current configurator can configure any protocol, here we better name it as `configurator`